ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -49,6 +49,17 @@ EOF
 
 function create_tests_temp_skip_list_13 {
     create_tests_temp_skip_list_12
```

Comment:
I think you need to remove this line then? Let's keep 12 and 13 sip lists separated from now.

Reply:
Removed